src/Symfony/Component/Form/Form.php

```diff
@@ -69,68 +69,41 @@
  */
 class Form implements \IteratorAggregate, FormInterface, ClearableErrorsInterface
 {
-    /**
-     * @var FormConfigInterface
-     */
-    private $config;
-
-    /**
-     * @var FormInterface|null
-     */
-    private $parent;
+    private FormConfigInterface $config;
+    private ?FormInterface $parent = null;
 
     /**
      * A map of FormInterface instances.
      *
      * @var FormInterface[]|OrderedHashMap
```

Comment:
the var annotation is somewhat wrong as it's not an array of FormInterface. this probably just for autocomplete reasons. better would be a template for OrderedHashMap. But that can be done independently.

Reply:
Yes, we should revisit a lot of those annotations, now that PhpStorm understands generics.